From d8734f498053ceaaf00ec36f45ec2c1976e76b2b Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Thu, 22 Oct 2020 16:51:57 +0800
Subject: [PATCH] arm64: rockchip_defconfig: Enable CONFIG_MMC_SDHCI_OF_DWCMSHC

DesignWare based SDHCI controller will be used for eMMC on some
Rockchip platforms such as RK3568.

Change-Id: I4a1c83f8a04c888d1279b1f5ef3aff34d602ea0f
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 3f87132818d8..39fd8b200f0e 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -747,6 +747,7 @@ CONFIG_MMC_BLOCK_MINORS=32
 CONFIG_MMC_SDHCI=y
 CONFIG_MMC_SDHCI_PLTFM=y
 CONFIG_MMC_SDHCI_OF_ARASAN=y
+CONFIG_MMC_SDHCI_OF_DWCMSHC=y
 CONFIG_MMC_DW=y
 CONFIG_MMC_DW_ROCKCHIP=y
 CONFIG_LEDS_GPIO=y
-- 
2.35.3

